ngraph_bridge/ngraph_utils.cc

```diff
@@ -586,6 +589,30 @@ void ClearAttribute(Graph* g,
   }
 }
 
+static ngraph::pass::PassConfig* _p_pass_config = nullptr;
```

Comment:
Singleton config is not required. You should check for enabled passes and attributes everytime `TranslateGraph()` is called.

Reply:
We can/may call PassEnabled() many times as needed. The singleton is to just read the env-var once at the very beginning. If we had a system-init kind of func, we could init it there too.